modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/package.json

```diff
@@ -13,7 +13,7 @@
 		"@clayui/table": "3.39.0",
 		"@liferay/map-google-maps": "*",
 		"@liferay/map-openstreetmap": "*",
-		"axios": "0.21.1",
+		"axios": "0.21.2",
```

Comment:
Why not upgrade to version 0.21.4 or even better the latest (0.24.0)?

According to the [changelog](https://github.com/axios/axios/blob/master/CHANGELOG.md) all **breaking changes** are related to typing things and once we have no TypeScript files using axios yet, I think we are safe to go to the latest.

Reply:
Because we don't need it. We're just trying to get rid of the vulnerability. If we bumped too much, I'm afraid it would require retesting the functionality.

Reply:
But hey... the functionality is not that big. We just have to make sure the progress bar is working during the file upload.
I updated the version.

Thanks for your inputs, buddy. 😄